src/db.jl

```diff
@@ -1,3 +1,5 @@
+using DataFrames
```

Comment:
Libraries should only be loaded into the namespace in the module declaration. Otherwise it can get weird with Julia. Also I think, we should try and not add `DataFrames` to the dependencies. I suspect that will negatively impact the compile times since `DataFrames.jl` is a big library with lots of dependencies. Is it critical to use this library?

Reply:
Not critical, `|> DataFrame` is just a convenient and idiomatic way of iterating over the results of a SQL query, and I figured that DataFrames was so common that it would be hard to avoid using it eventually. But we can get rid of DataFrames and iterate over results another way (the key is to make sure we actually completely iterate over all results).

Reply:
Let's try it with DataFrames and see whether it significantly impacts compile times. If not, it should be fine, it is more convenient this way.